### ISSUE TITLE
Rendering midi with animation

### ISSUE THREAD

**Issue by osvi64** [paulrosen/abcjs#125]
Hi
I would like to ear midi sound while seeing animation and cursor on the notes played.
Which is the best way to do it? I see that in ABCJS.renderMidi function there is a parameter called "animate" that, I guess, will do something similar; but I didn't understand how to use it, sorry.
If this is the way to go, please can you provide an example on how to use it?
Else can you suggest which is the way to go?

I would appreciate if you can help me. Thank you in advance anyway.

**Comment by paulrosen** [comment]
Here is an example: https://github.com/paulrosen/vue-abcjs-midi-demo/blob/master/src/components/HelloWorld.vue

**Comment by osvi64** [comment]
Hi Paul

Thank you for the answer. I was able to run the example after some time because I don't know the vue technology, so it was not immediate for me. It works as expected but it doesn't cover totally my needs.
As far as I understood it works interactively: I can modify the abc source and I can start, pause and stop the midi player using the midi control.

Can I do the same programmatically, without using the editor and the midi controls? I tried hiding the textarea field, the midi control and simulating the click on the midi controls via js. 

It works but it doesn't sound to me as the best solution; is there a way to do it calling directly some primitive functions  (maybe ABCJS.renderAbc or ABCJS.renderMidi )? 

Thank you
Osvaldo
